apps/backend/api/src/routes/patient.route.ts

```diff
@@ -129,8 +129,7 @@ export const relatedFamilyMemberSchema: z.ZodType = (
 
 patientRouter.post(
   '/:id/familyMember/:familyMemberId',
-  paramsParser(),
-  validateSplit(relatedFamilyMemberSchema, undefined, undefined),
+  validateSplit(undefined, undefined, relatedFamilyMemberSchema),
```

Comment:
explain swap?

Reply:
Well, since it's a post request, it would make sense to use the body. But yes, in this case, I should change the URL too

Reply:
https://www.mscharhag.com/api-design/rest-many-to-many-relations